package.json

```diff
@@ -101,6 +101,7 @@
     "babel-preset-expo": "8.2.3",
     "eslint": "6.7.2",
     "eslint-config-prettier": "6.12.0",
+    "eslint-plugin-import": "^2.22.1",
```

Comment:
No ^ so we all have the same version

Reply:
Everyone should have the same version even when using ^ because the actual version to be installed when running yarn install is the version from the yarn.lock file. This makes it easier to upgrade packages later, and allows you to lock a version when really needed.

Reply:
In my experience, it's easier to debug without ^, I can know exactly which version I'm using without having a look into the yarn.lock file

Reply:
In case you have to debug a exact version you should use `yarn why`, this way you you can check which version of a package is included, why it is included, and even if there are multiple versions.